extension/content.js

```diff
@@ -51,6 +52,24 @@ function addReleasesTab() {
 	}
 }
 
+function addBlameParentLinks() {
+	let pathname = location.pathname;
+
+	$('.blame-sha').each(function(index, commitLink) {
+		let $commitLink = $(commitLink)
+		let $blameParentLink = $commitLink.clone();
+		let commitSha = commitLink.href.match(/(?<=commit\/)\w+/)[0];
```

Comment:
I've removed the lookbehinds. Take a look at the new regular expressions. I've used a group around `\w+`, not around `commit`. I could add the `{40}`, but not sure if it won't do more harm than good. Git and GitHub accept only the beginning of SHA hashes as well. I definitely don't want to limit it with the `$`. I don't want to assume there won't be anything added later to the path.